lib/ex_aws/utils.ex

```diff
@@ -100,4 +100,15 @@ defmodule ExAws.Utils do
 
     greg_secs - @seconds_0_to_1970
   end
+
+  def uuid, do: DateTime.utc_now |> :erlang.phash2
+
+  def rename_keys(keywords, mapping) do
+    keywords |> Enum.map(fn {k, v} ->
+     cond do
+       Keyword.has_key?(mapping, k) -> {mapping[k], v}
+       true -> {k, v}
+     end
+    end)
+  end
```

Comment:
I think this would be better as:
```
def rename_keys(params, mapping) do
  mapping = Map.new(mapping)
  Enum.map(params, fn {k, v} ->
    {Map.get(mapping, k, k), v}
  end)
end
```